content/en/docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -36,19 +36,32 @@ port: 20250
 serializeImagePulls: false
 evictionHard:
     memory.available:  "200Mi"
+    nodefs.available:  "10%"
+    imagefs.available: "15%"
+    nodefs.inodesFree: "5%"
```

Comment:
Why not use the default 100Mi for memory.available?

```
	"memory.available":  "100Mi",
	"nodefs.available":  "10%",
	"nodefs.inodesFree": "5%",
	"imagefs.available": "15%",
```
Why not use alphabetic order?

Reply:
That seems good